src/disco/components/InstallButton.js

```diff
@@ -41,6 +43,28 @@ export class InstallButton extends React.Component {
     downloadProgress: 0,
   }
 
+  getLabel() {
+    const { i18n, name, status, type } = this.props;
+    if (status === DOWNLOADING) {
+      return i18n.sprintf(i18n.gettext('Downloading %(name)s'), { name });
+    } else if (status === INSTALLING) {
+      return i18n.sprintf(i18n.gettext('Installing %(name)s'), { name });
+    } else if ([ENABLED, INSTALLED].includes(status) && type === THEME_TYPE) {
+      return i18n.sprintf(
+        i18n.gettext('%(name)s is installed and enabled. Click to disable'), { name });
```

Comment:
We uninstall themes as well, so we should be able to use the `Click to uninstall` copy for both.